.github/workflows/examples-efr32.yaml

```diff
@@ -15,102 +15,121 @@
-                        --enable-flashbundle \
-                        --target-glob '*-brd4161a-{lock,light,switch,window-covering,unit-test}' \
-                        build \
-                        --copy-artifacts-to out/artifacts \
-                     "
-            - name: Prepare some bloat report from the previous builds
-              run: |
-                  .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py \
-                     efr32 BRD4161A lighting-app \
-                     out/efr32-brd4161a-light/chip-efr32-lighting-example.out \
-                     /tmp/bloat_reports/
-                  .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py  \
-                     efr32 BRD4161A window-app \
-                     out/efr32-brd4161a-window-covering/chip-efr32-window-example.out \
-                     /tmp/bloat_reports/
-            - name: Build example EFR32 Lighting App for BRD4161A with RPCs
-              timeout-minutes: 10
-              run: |
-                  scripts/examples/gn_efr32_example.sh examples/lighting-app/efr32/ out/lighting_app_debug_rpc BRD4161A \
-                    'import("//with_pw_rpc.gni")'
-                  .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py efr32 BRD4161A+rpc lighting-app \
-                    out/lighting_app_debug_rpc/BRD4161A/chip-efr32-lighting-example.out /tmp/bloat_reports/
-            - name: Build example EFR32+WF200 WiFi Lock app for BRD4161A
-              timeout-minutes: 10
-              run: |
-                  scripts/examples/gn_efr32_example.sh examples/lock-app/efr32/ out/lock_app_wifi_wf200 BRD4161A is_debug=false --wifi wf200
-                  .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py efr32 BRD4161A+wf200 lock-app \
-                    out/lock_app_wifi_wf200/BRD4161A/chip-efr32-lock-example.out /tmp/bloat_reports/
-            - name: Build example EFR32+RS911x WiFi Lighting app for BRD4161A
-              timeout-minutes: 10
-              run: |
-                  scripts/examples/gn_efr32_example.sh examples/lighting-app/efr32/ out/lighting_app_wifi_rs911x BRD4161A --wifi rs911x
-                  .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py efr32 BRD4161A+rs911x lighting-app \
-                    out/lighting_app_wifi_rs911x/BRD4161A/chip-efr32-lighting-example.out /tmp/bloat_reports/
-            - name: Uploading Size Reports
-              uses: actions/upload-artifact@v2
-              if: ${{ !env.ACT }}
-              with:
-                  name: Size,EFR32-Examples,${{ env.GH_EVENT_PR }},${{ env.GH_EVENT_HASH }},${{ env.GH_EVENT_PARENT }},${{ github.event_name }}
-                  path: |
-                      /tmp/bloat_reports/
+      - name: Build some BRD4161A variants
+        timeout-minutes: 50
+        run: |
+          ./scripts/run_in_build_env.sh \
+             "./scripts/build/build_examples.py \
```

Comment:
Will build_examples.py require to be run only after lfs updates? please make a check/run in the efr32.py logic then. Developers should be able to just compile efr32 builds without worrying about each platform particularities.